playground/index.html

```
@@ -1,89 +1,167 @@
+					Static checker for GitHub Actions workflow files
+				</h2>
+			</header>
+			<div id="controls" class="navbar-end">
+				<div class="control">
+					<a id="permalink-btn" class="button">Permalink</a>
+				</div>
+				<div class="field">
+					<div id="url-input" class="field has-addons">
+						<div class="control">
+							<input
+								id="check-url-input"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)